server/server/managers.py

```diff
@@ -0,0 +1,125 @@
+import asyncio
+import json
+import uuid
+from typing import Dict, List
+
+from fastapi import WebSocket
+
+import server.user as user
+
+
+class DbManager:
+    """Manages the Database operations"""
+
+    def __init__(self, user_db_file: str, rooms_db_file: str):
+        self.user_db_file = user_db_file
+        self.rooms_db_file = rooms_db_file
+        u = open(user_db_file)
+        r = open(rooms_db_file)
+        self.users = json.loads(u.read())
+        self.rooms = json.loads(r.read())
+        u.close()
+        r.close()
```

Comment:
use context manager instead, 
like 
```suggestion
        with open(user_db_file) as DB_FILE, open(other_file) as OTHER_FILE :
            # do operation with file
```
no need to close the file as when the `with` statement is over all the files will be closed automatically

Reply:
oh i forgot about this, i will do it